msbuild/Xamarin.MacDev.Tasks.Core/Tasks/DSymUtilTaskBase.cs

```diff
@@ -62,7 +62,7 @@ protected override string GenerateCommandLineCommands ()
 		{
 			var args = new CommandLineBuilder ();
 
-			args.AppendSwitch ("-t");
+			args.AppendSwitch ("-num-threads");
 			args.AppendSwitch ("4");
 			args.AppendSwitch ("-z");
```

Comment:
Looks like that `-z` is not documented - so we risk running in the same issue soon.

Reply:
I would argue that in this scenario the switch would be entirely ignored because it does not take a parameter but yup nice catch

Reply:
That said `-z` switch does show for me: `xcrun dsymutil -help ` last documented switch option

```
-z                      Alias for --minimize
```

Reply:
does not show w/Xcode 11.3 - but it's fine if it's documented under 11.4